src/LiveComponent/src/Attribute/LivePropContext.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\UX\LiveComponent\Attribute;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ *
+ * @experimental
+ *
+ * @internal
+ */
+final class LivePropContext
+{
+    private LiveProp $liveProp;
+    private \ReflectionProperty $reflectionProperty;
+
+    public function __construct(LiveProp $liveProp, \ReflectionProperty $reflectionProperty)
+    {
+        $this->liveProp = $liveProp;
+        $this->reflectionProperty = $reflectionProperty;
+    }
+
+    public function liveProp(): LiveProp
+    {
+        return $this->liveProp;
+    }
+
+    public function reflectionProperty(): \ReflectionProperty
```

Comment:
This is a nit pick, but I noticed that we only ever need the `->getName()` off of this - we could just pass in that string.

Reply:
We do the actual `\ReflectionProperty` [here](https://github.com/symfony/ux/pull/106/files#diff-b6ca1bfe380a148203d16367e6f7ae91a700b7d7b38b0d34114d4995c9eeafefR227).

Reply:
Ah, ok I missed it 👍